ocs_ci/ocs/resources/mcg.py

```
@@ -822,31 +822,43 @@ def check_if_mirroring_is_done(self, bucket_name, timeout=300):
+                                "key": obj_key,
+                                "obj_id": written_object.get("obj_id"),
+                            },
+                        )
+                        .json()
+                        .get("reply")
+                        .get("chunks", [])
+                    )
+                except CommandFailed:
+                    logger.warning(
+                        f"Transient RPC error reading object mapping for "
+                        f"'{obj_key}', skipping this object"
```

Ruff (0.15.13)

[warning] Logging statement uses f-string

(G004)